.husky/pre-commit

```diff
@@ -2,8 +2,9 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx is-ci && exit 0
+cd packages/askui-nodejs
 npm run lint:staged
 npm test
 
-cd examples/askui-nodejs-base-example || exit
+cd ../../examples/askui-nodejs-base-example || exit
```

Comment:
Save the root directory into a variable, e.g., `project_root=$(pwd)`, and use it so that you don't have to go up and down the file system tree using relative paths but you use absolute paths instead, e.g., `${project_root}/examples/askui-nodejs-base-example`. This makes these `cd` commands independent of each other and, therefore, omits a lot of potential bugs.

Reply:
Sometimes I think too complicated to come up with such an efficient solution 😉